docker-images/artifacts/kafka-thirdparty-libs/4.1.x/pom.xml

```diff
@@ -19,7 +19,7 @@
         <strimzi-oauth.version>0.17.0</strimzi-oauth.version>
         <strimzi-metrics-reporter.version>0.2.0</strimzi-metrics-reporter.version>
         <prometheus.version>1.3.6</prometheus.version>
-        <cruise-control.version>2.5.142</cruise-control.version>
+        <cruise-control.version>2.5.145</cruise-control.version>
```

Comment:
@ppatierno Was there any particular reason why Kafka 4.1.0 was using 2.5.142 and not 2.5.143 that we need to check? Or was it just some mistake?

Reply:
Is there any specific relationship between Kafka 4.1 and 2.5.145 that we should have updated it? What kind of mistake are you talking about? I don't think there is nothing to check for using CC 2.5.145, so using the latest is ok.